ql/src/java/org/apache/hadoop/hive/ql/exec/vector/VectorAggregationDesc.java

```diff
@@ -126,6 +130,18 @@ public VectorAggregationDesc(String aggregationName, GenericUDAFEvaluator evalua
     this.vecAggrClass = vecAggrClass;
   }
 
+  public VectorAggregationDesc(String aggregationName, GenericUDAFEvaluator evaluator,
```

Comment:
Could we logically group parameters into their own higher-level object/use Builder pattern?